internal/featuresupport/features.go

```diff
@@ -16,6 +16,7 @@ const (
 )
 
 type SupportLevelFeature interface {
+	New() SupportLevelFeature
```

Comment:
please add a comment about why you added it to the interface

Reply:
I think it will be better to call this method - dependencies - it might be more than one feature

Reply:
This method basically create the feature structure with it's default attributes. Not sure the name dependencies is right here

Reply:
I also documented the entire interface